### DISCUSSION TITLE
Target #114 updated

### DISCUSSION THREAD

**Discussion by chinchang** [cssbattle/forum#273] [Announcements]
Players, original target #114 was a little too strict in terms of scoring and it was very difficult to get 100% match on it. So we have now tweaked it a bit to make 100% match achievable on it. Please rework on your code and submit again.

**Note**: you might have to hard refresh to see the new target -> https://cssbattle.dev/play/114 

Apologies for the time spent on trying to get 100% match on it 🙇🏼 🙏🏼

**Comment by rafehimself** [comment]
don't you guys check is it solvable or there is no bugs before releasing a target? 😃

**Reply by chinchang** [reply]
Yes, sometimes we miss due to constraints! 😓
